public/js/encryptionManager.js

```diff
@@ -118,17 +122,30 @@ EncryptionManager.prototype.loadMasterKeyPair = function loadMasterKeyPair(room,
   var self = this;
   if (masterKeyPair) {
     // MasterKey mode
-    console.log("[ENCRYPTION MANAGER] masterKeyPair found! client keyManager locked: "+self.keyManager.is_pgp_locked().toString());
-    if (self.keyManager.is_pgp_locked()) { return console.log("[ENCRYPTION MANAGER] (loadMasterKeyPair) Client keyManager is locked! :(") };
-    if (!masterKeyPair.encryptedPrivateKey) { return console.log("[ENCRYPTION MANAGER] (loadMasterKeyPair) No master key provided to loadMasterKeyPair! encryptedMasterPrivateKey is NULL") };
+    console.log("[ENCRYPTION MANAGER] masterKeyPair found! client keyManager locked", self.keyManager.is_pgp_locked().toString());
+
+    if (self.keyManager.is_pgp_locked()) {
+      return console.log("[ENCRYPTION MANAGER] (loadMasterKeyPair) Client keyManager is locked! :(");
+    }
+    if (!masterKeyPair.encryptedPrivateKey) {
+      return console.log("[ENCRYPTION MANAGER] (loadMasterKeyPair) No master key provided to loadMasterKeyPair! encryptedMasterPrivateKey is NULL");
+    }
+
     // Decrypt master key and add to keyRing
     console.log("[ENCRYPTION MANAGER] (loadMasterKeyPair) Decrypting master key");
+
     self.decryptMasterKey(masterKeyPair.encryptedPrivateKey, function(err, masterPrivateKey) {
-      window.encryptionManager.getKeyManager({ publicKey: masterKeyPair.publicKey, privateKey: masterPrivateKey, passphrase: '' }, function(err, keyManager) {
+      window.encryptionManager.getKeyManager({
+        publicKey: masterKeyPair.publicKey,
+        privateKey: masterPrivateKey,
+        passphrase: ''
+      }, function(err, keyManager) {
         self.masterKeyManager = keyManager;
         // Unlock and add masterKeyManager to keyRing
         window.encryptionManager.unlockMasterKey(room, function(err) {
-          if (err) { return callback(err, false) };
+          if (err) {
+            return callback(err, false);
+          }
```

Comment:
Sections like this you had semicolons after the end bracket for the if statement rather than after the logic. For readability it's also usually good to avoid single line if statements.